.github/CODEOWNERS

```diff
@@ -1423,10 +1423,77 @@ x-pack/test/api_integration/deployment_agnostic/services/ @elastic/appex-qa
+/test/server_integration/http @elastic/kibana-core
+/test/scripts/run_multiple_kibana_nodes.sh @elastic/kibana-core
+/test/plugin_functional/test_suites/application_links @elastic/kibana-core
+/test/interpreter_functional/test_suites/run_pipeline @elastic/kibana-core
+/test/functional/services/usage_collection.ts @elastic/kibana-core
+/test/functional/services/saved_objects_finder.ts @elastic/kibana-core
+/test/api_integration/fixtures/import_managed* @elastic/kibana-core
+/test/functional/services/apps_menu.ts @elastic/kibana-core
+/x-pack/test/functional/apps/status_page @elastic/kibana-core
+/x-pack/test/cloud_integration @elastic/kibana-core
 /x-pack/test/cloud_integration/plugins/saml_provider @elastic/kibana-core
-/x-pack/test/functional_embedded/plugins/iframe_embedded @elastic/kibana-core
+/test/server_integration/plugins/status_plugin_* @elastic/kibana-core
+/x-pack/test/functional_cors @elastic/kibana-core
+/x-pack/test/stack_functional_integration/apps/telemetry @elastic/kibana-core
+/test/plugin_functional/test_suites/telemetry @elastic/kibana-core
+/test/plugin_functional/test_suites/usage_collection @elastic/kibana-core
+/test/plugin_functional/test_suites/saved_objects* @elastic/kibana-core
+/test/plugin_functional/test_suites/core* @elastic/kibana-core
+/test/plugin_functional/plugins/telemetry @elastic/kibana-core
+/test/plugin_functional/plugins/session_notifications @elastic/kibana-core
+/test/plugin_functional/plugins/kbn_top_nav/ @elastic/kibana-core
+/test/plugin_functional/plugins/elasticsearch_client_plugin @elastic/kibana-core
+/test/plugin_functional/plugins/app_link_test @elastic/kibana-core
+/x-pack/test/plugin_functional/plugins/global_search_test @elastic/kibana-core
+/test/plugin_functional/plugins/saved_object* @elastic/kibana-core
+/test/plugin_functional/plugins/rendering_plugin @elastic/kibana-core
+/test/interpreter_functional/plugins/kbn_tp_run_pipeline @elastic/kibana-core
+/x-pack/test/functional/fixtures/kbn_archiver/saved_objects_management @elastic/kibana-core
+/x-pack/test/functional_embedded @elastic/kibana-core
+/test/plugin_functional/plugins/core* @elastic/kibana-core
+/test/node_roles_functional @elastic/kibana-core
+/test/functional/page_objects/newsfeed_page.ts @elastic/kibana-core # assigned per https://github.com/elastic/kibana/pull/160210
+/test/functional/page_objects/home_page.ts @elastic/kibana-core
+/test/functional/fixtures/es_archiver/deprecations_service @elastic/kibana-core
+/test/health_gateway/**/* @elastic/kibana-core
+/test/api_integration/apis/saved_objects* @elastic/kibana-core
+/test/functional/apps/home @elastic/kibana-core
+/test/health_gateway @elastic/kibana-core
+/test/node_roles_functional @elastic/kibana-core
+/test/plugin_functional/plugins/app_link_test @elastic/kibana-core
+/test/plugin_functional/plugins/kbn_top_nav @elastic/kibana-core
+/test/plugin_functional/plugins/elasticsearch_client_plugin @elastic/kibana-core
```

Comment:
nit: Would you mind grouping these with the other `plugin_functional`?

Reply:
For sure, but what do you mean by grouping them Jean-Louis?

Reply:
I just meant placing them next to the other ones, nothing fancy :)

Reply:
Also, I noticed some duplication; dropped: https://github.com/elastic/kibana/pull/199771/commits/7227d8aec26c3b5117eca2e909a5697b4fada0e5

Reply:
Grouped them together as asked: https://github.com/elastic/kibana/pull/199771/commits/2ce91e999b3ea5a1a59e862d9790dd8f76bdd9c3